src/txmempool.h

```diff
@@ -360,7 +360,7 @@ class CTxMemPool
      * check does nothing.
      */
     void check(const CCoinsViewCache *pcoins) const;
-    void setSanityCheck(double dFrequency = 1.0) { nCheckFrequency = dFrequency * 4294967296.0; }
+    void setSanityCheck(double dFrequency = 1.0) { nCheckFrequency = dFrequency * 4294967295.0; }
```

Comment:
Agree w/ @dcousens 

Additionally, raw big numbers with no explanation comment should be avoided.  I know it's 2^32 but it's not immediately obvious to every code reader.

This should be uint_max-1 and getdouble() should be similar.